main.py

```diff
@@ -45,6 +48,8 @@ def process_message_event(client, event):
             return
         user = event['user']
         if any(hit in text.lower() for hit in HITS):
+            if any(excludeUser.lower() in user.lower() for excludeUser in EXCLUDE_USERS):
```

Comment:
We could exit out even earlier in the function, since we don't need to check if they said something that triggers the bot's response, since it'll be faster and avoid the other work. I'd also suggest keeping the excluded users in a Python set instead of array, then we can just check `if user.lower() in EXCLUDE_USERS`.